model/pom.xml

```diff
@@ -41,7 +41,7 @@
     <dependency>
       <groupId>com.graphaware.neo4j</groupId>
       <artifactId>uuid</artifactId>
-      <version>2.2.6.35.7</version>
+      <version>3.3.3.52.16</version>
```

Comment:
was there another issue with building that necessitated this version change? just wondering, I don't think I ran into any issues with only changing the `com.graphaware.neo4j:server-community` dependency version above